src/lib/cat_superhero.svelte

```diff
@@ -381,23 +497,84 @@
     }
     /* Roadmap, markers, and text */
     .roadmap {
-      background-image: url(@base/roadmap/road.svg);
+      display: block;
+      background-image: none;
       background-repeat: no-repeat;
       background-size: auto 100%;
-      padding-top: 6rem;
+      padding-top: 18rem;
       padding-bottom: 5vw;
       margin: 0 1rem;
+      position: relative;
+    }
+
+    .toggle-button {
+      width: 0;
+      height: 0;
+      border-left: 10px solid transparent;
+      border-right: 10px solid transparent;
+      border-top: 10px solid var(--color-secondary-accent);
+      transition: transform 0.2s ease-in;
     }
+
+    .toggle-button[aria-expanded='true'] {
+      transform: scaleY(-1);
```

Comment:
I think a rotation might be better here, but I'm nitpicking.

```suggestion
      transition: transform 0.2s;
    }

    .toggle-button[aria-expanded='true'] {
      transform: rotate(180deg);
```